.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,13 @@
+_Read these instructions before submitting_
+
 **DELETE THIS TEMPLATE BEFORE SUBMITTING**
 
 _Only use Github issues to report bugs or feature requests, see
 https://www.packer.io/community.html_
 
+Specifically we _don't accept_ issues for:
+- _Timeout waiting for SSH/WinRM_ - Unless there is evidence pointing towards a bug in packer. e.g it works with a previous version, or you can manually connect with the same credentials, etc. Ask on the mailing list if you are unsure.
+
```

Comment:
How about we soften the tone by saying something like "For example,Timeouts waiting for SSH/WinRM are generally not bugs within packer and are better addressed by the mailing list" instead of having this highly structured "we do not allow this" format.